src/particles/deposition/ExplicitDeposition.cpp

```diff
@@ -161,20 +161,23 @@ ExplicitDeposition (PlasmaParticleContainer& plasma, Fields& fields, const Multi
                         [[maybe_unused]] auto laser_fac = a_laser_fac;
                         [[maybe_unused]] auto clight = a_clight;
                         if constexpr (use_laser.value) {
```

Comment:
Just to remove one level if indentation?
```suggestion
                        if constexpr (use_laser.value && shape_x * shape_y != 0._rt) {
```

Reply:
This won’t work because `shape_x * shape_y != 0._rt` is not known at compile time

Reply:
Oh, right, that's the reason